torch/quantization/quantize.py

```diff
@@ -220,6 +226,16 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     convert(model)
     return model
 
+def quantize_dynamic(model, run_fn=None, run_args=None, qconfig_dict=None):
+    r"""Converts a float model to dynamic quantized model. Do dynamic training and output a quantized model.
+    """
+    model.eval()
+    model = prepare_dynamic(model, qconfig_dict)
+    if run_fn is not None:
+        run_fn(model, run_args)
+    convert(model, DEFAULT_DYNAMIC_MODULE_MAPPING)
```

Comment:
pass DEFAULT_DYNAMIC_MODULE_MAPPING as arguments too

Reply:
Will submit a PR on top of #23212 for this change.

Reply:
DEFAULT_DYNAMIC_MODULE_MAPPING is now the default value for an argument `mapping` in the `quantize_dynamic` function.